packages/site/pages/components/starrating.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+
+import * as Text from '@pluralsight/ps-design-system-text/react'
+import StarRating from '@pluralsight/ps-design-system-starrating/react'
+
+import {
+  Chrome,
+  Code,
+  Content,
+  Example,
+  P,
+  PageHeading,
+  PropTypes,
+  SectionHeading,
+  withServerProps
+} from '../../src/ui'
+
+export default withServerProps(_ => (
+  <Chrome>
+    <Content title="StarRating">
+      <PageHeading packageName="starrating">StarRating</PageHeading>
+
+      <P>Install the component dependency:</P>
+      <Code language="bash">
+        npm install @pluralsight/ps-design-system-starrating
+      </Code>
+
+      <P>Include a React component in your project:</P>
+      <Code language="javascript">
+        import StarRating from '@pluralsight/ps-design-system-starrating/react'
+      </Code>
+
+      <PropTypes
+        props={[
+          PropTypes.row([
+            'onChange',
+            'function',
```

Comment:
Most of the time, we've tried to provide a signature for these where data will be passed back.  

Some examples
https://github.com/pluralsight/design-system/blob/master/packages/site/pages/components/actionmenu.js#L88
https://github.com/pluralsight/design-system/blob/master/packages/site/pages/components/datepicker.js#L79